sma_multi_test.go

```diff
@@ -0,0 +1,61 @@
+package movavg_test
+
+import (
+	. "github.com/blocktop/movavg"
```

Comment:
In Go world it makes sense to keep the original addr when PR-ing :) You can have a package in `$GOPATH/github.com/mxmCherry/movavg`, but have `git add my git@github.com/blocktop/movavg.git && git push origin my branch-name`